eng/pipelines/libraries/build-test-job.yml

```diff
@@ -54,15 +53,12 @@ jobs:
             artifactFileName: $(librariesBuildArtifactName)$(archiveExtension)
             unpackFolder: $(Build.SourcesDirectory)/artifacts
             cleanUnpackFolder: false
-        
-        - script: $(_buildScript) -restore $(_buildArguments) $(_skipTestRestoreArg)
-          displayName: Restore
 
         - script: $(_buildScript)
-                -buildtests
+                -subset libtests
```

Comment:
Nice, so with this we will only build and restore tests?

Reply:
right :)